### PR TITLE
Veoss 402 - Added Submitted returns page

### PR DESCRIPTION
**Pull request by warrenskiwork1979**

### CODE REVIEW

File: test/controllers/SubmittedReturnsHistoryControllerSpec.scala [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+          bind[VatReturnConnector].toInstance(vatReturnConnector)
+        ).build()
+
+      when(vatReturnConnector.get(any())(any())) thenReturn Future.successful(Right(completeVatReturn))
+
+      running(application) {
+        val request = FakeRequest(GET, routes.SubmittedReturnsHistoryController.onPageLoad().url)
+
+        val result = route(application, request).value
+        println("Result for Right: " + result)
```
Comment (DevAndyroo) [inline test/controllers/SubmittedReturnsHistoryControllerSpec.scala:59]: can we remove this?

**Review by DevAndyroo — CHANGES_REQUESTED** [review]

File: app/views/SubmittedReturnsHistoryView.scala.html [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        <ul class="govuk-tabs__list">
+            <li class="govuk-tabs__list-item govuk-tabs__list-item--selected">
+                <a class="govuk-tabs__tab" href="#2021">
+                    2021
+                </a>
+            </li>
+        </ul>
+        <div class="govuk-tabs__panel" id="returns_year_2021">
+            <h2 class="govuk-heading-m">2021 returns</h2>
+            @if(vatReturn.isDefined) {
```
Comment (DevAndyroo) [inline app/views/SubmittedReturnsHistoryView.scala.html:37]: We can also use a match statement here,

```
@maybeVatReturn match {
  case Some(vatReturn) => {
    <table class="govuk-table">
                <ul class="govuk-list govuk-list--bullet">
                    <li>
                        <a class="govuk-link" href="@routes.PreviousReturnController.onPageLoad(vatReturn.period)">@vatReturn.period.displayText</a>
                    </li>
                </ul>
            </table>
  }
  case _ => {
    <p class="govuk-body-m">@messages("submittedReturnsHistory.no-returns")</p>
  }

}
```

 which in my opinion is cleaning as we would clear up using `.get`s everytime we need to access

File: test/controllers/SubmittedReturnsHistoryControllerSpec.scala (hunk 1 above)
Comment (ansahdavid) [reply test/controllers/SubmittedReturnsHistoryControllerSpec.scala:59]: Yep done now

File: app/views/SubmittedReturnsHistoryView.scala.html (hunk 2 above)
Comment (ansahdavid) [reply app/views/SubmittedReturnsHistoryView.scala.html:37]: Yep, like this recommendation cheers

**Review by DevAndyroo — APPROVED** [review]

**Pull request merged**